core/src/test/kotlin/de/fraunhofer/iem/spha/core/strategy/XorKPICalculationStrategyTest.kt

```diff
@@ -0,0 +1,298 @@
+package de.fraunhofer.iem.spha.core.strategy
+
+import de.fraunhofer.iem.spha.core.hierarchy.KpiHierarchyNode
+import de.fraunhofer.iem.spha.model.kpi.KpiId
+import de.fraunhofer.iem.spha.model.kpi.KpiStrategyId
+import de.fraunhofer.iem.spha.model.kpi.RawValueKpi
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiCalculationResult
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiEdge
+import de.fraunhofer.iem.spha.model.kpi.hierarchy.KpiNode
+import kotlin.test.Test
+import org.junit.jupiter.api.Assertions.*
+
+class XorKPICalculationStrategyTest {
+
+    @Test
+    fun emptyEdges() {
+        assertEquals(
+            KpiCalculationResult.Empty(),
+            XorKPICalculationStrategy.calculateKpi(listOf(), strict = true),
+        )
+        assertEquals(
+            KpiCalculationResult.Empty(),
+            XorKPICalculationStrategy.calculateKpi(listOf(), strict = false),
+        )
+    }
+
+    @Test
+    fun toManyEdges() {
```

Comment:
I think it's missing one `o`.
```suggestion
    fun tooManyEdges() {
```